### PR TITLE
Image export

### PR DESCRIPTION
**Pull request by robinschmid**
added options to handle missing values and missing spectra in images. Missing spectra might come from irregular shapes during image acquisition.

### CODE REVIEW

File: src/main/java/io/github/mzmine/modules/io/export_image_to_csv/ImageToCsvExportParameters.java [hunk 1]
```diff
@@ -20,18 +20,46 @@
...
-    super(new Parameter[]{dir, delimiter});
+    super(new Parameter[]{dir, delimiter, handleMissingSpectra, handleMissingSignals});
+  }
+
+  /**
+   * Options to handle missing values due to irregular shapes during image acquisition (no scan at
+   * specific x,y coordinate) and missing signals in available scans.
+   */
+  public enum HandleMissingValues {
+    // leave empty in csv means ,,
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/io/export_image_to_csv/ImageToCsvExportParameters.java:55]: use actual javadoc here

File: src/main/java/io/github/mzmine/modules/io/export_image_to_csv/ImageToCsvExportParameters.java [hunk 2]
```diff
@@ -20,18 +20,46 @@
...
+    LEAVE_EMPTY,
+    // replace by zero or by lowest value in image
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/io/export_image_to_csv/ImageToCsvExportParameters.java:57]: s.o.

**Review by SteffenHeu — COMMENTED** [review]
Only minor comments

**Comment by robinschmid** [comment]
Updated javadoc - ready to merge from my side

**Pull request merged**